### PR TITLE
chore: Update dependencies

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,21 +29,21 @@
...
+    "@babel/preset-env": "^7.10.4",
     "babel-jest": "^25.1 || ^26",
     "eslint": "^6 || ^7",
     "eslint-config-airbnb-base": "^14.2.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-jest": "^23.17.1",
     "eslint-plugin-prettier": "^3.1.4",
-    "execa": "^2.1.0",
+    "execa": "^3.4.0",
```
Comment (SimenB) [inline package.json:42]: for some reason I though v3 dropped node 8. seems not!

File: package.json [hunk 2]
```diff
@@ -29,21 +29,21 @@
...
     "jest": "^25.1 || ^26",
     "jest-watch-select-projects": "^2.0.0",
     "jest-watch-typeahead": "^0.5.0",
-    "prettier": "^1.14.3",
+    "prettier": "1.19.1",
```
Comment (SimenB) [inline package.json:46]: we don't need to do this since we have a lockfile. doesn't hurt either, tho

File: package.json [hunk 3]
```diff
@@ -29,21 +29,21 @@
     "create-jest-runner": "^0.6.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.1.0",
-    "@babel/core": "^7.1.0",
-    "@babel/preset-env": "^7.1.0",
+    "@babel/cli": "^7.10.4",
```
Comment (SimenB) [inline package.json:32]: ?

Comment (MichaelDeBoey) [reply package.json:32]: Just to make it the highest possible, that's all

File: package.json (hunk 1 above)
Comment (MichaelDeBoey) [reply package.json:42]: Nope v4 drops Node 8

**Comment by SimenB** [comment]
Thanks!

**Pull request merged**